src/Command/SetSettingsCommand.php

```diff
@@ -0,0 +1,182 @@
+            ->addOption(self::OPTION_CHANNEL, 'c', InputOption::VALUE_OPTIONAL, 'Channel code')
+            ->addOption(self::OPTION_LOCALE, 'l', InputOption::VALUE_OPTIONAL, 'Locale code')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        try {
+            /** @var string $alias */
+            $alias = $input->getArgument(self::ARGUMENT_ALIAS);
+            /** @var string $path */
+            $path = $input->getArgument(self::ARGUMENT_PATH);
+            $channelCode = $input->getOption(self::OPTION_CHANNEL);
+            /** @var ?string $locale */
+            $locale = $input->getOption(self::OPTION_LOCALE);
+
+            $channel = null;
+            if (null !== $channelCode) {
+                /** @var ?ChannelInterface $channel */
+                $channel = $this->channelRepository->findOneBy(['code' => $channelCode]);
+            }
+
+            /** @var SettingsInterface $settings */
+            $settings = $this->settingsRegistry->getByAlias($alias);
+            ['vendor' => $vendor, 'plugin' => $plugin] = $settings->getAliasAsArray();
+            $setting = $this->getSetting($vendor, $plugin, $path, $locale, $channel);
+
+            /** @var string $type */
+            $type = $input->getArgument(self::ARGUMENT_TYPE);
+            $this->validateType($type);
+
+            $value = $input->getArgument(self::ARGUMENT_VALUE);
+
+            $setting->setStorageType($type);
+            /** @phpstan-ignore-next-line */
+            $setting->setValue($this->settingsFormatter->formatValue($type, $value));
+
+            $this->settingManager->persist($setting);
+            $this->settingManager->flush();
+        } catch (Exception $e) {
+            $output->writeln(sprintf('<error>%s</error>', $e->getMessage()));
+
+            return Command::FAILURE;
+        }
+
+        $output->writeln(sprintf('<info>%s</info>', 'The setting has been saved'));
+
+        return Command::SUCCESS;
+    }
+
+    private function getSetting(string $vendor, string $plugin, ?string $path, ?string $locale, ?ChannelInterface $channel): SettingInterface
+    {
+        /** @var SettingInterface|null $setting */
+        $setting = $this->settingRepository->findOneBy([
+            'vendor' => $vendor,
+            'plugin' => $plugin,
+            'path' => $path,
+            'localeCode' => $locale,
+            'channel' => $channel,
+        ]);
+
+        // Reset existing value
+        if ($setting) {
+            $setting->setValue(null);
+        }
+
+        if (null === $setting) {
+            /** @var SettingInterface $setting */
+            $setting = $this->settingFactory->createNew();
+            $setting->setVendor($vendor);
+            $setting->setPlugin($plugin);
+            $setting->setPath($path);
+            $setting->setLocaleCode($locale);
+            $setting->setChannel($channel);
+        }
+
+        return $setting;
+    }
```

Comment:
**🎯 suggestion:** ‏This code seems to be a duplicate of the Fixtures, maybe we could have a specific service for it and use it in both places?